values.yaml

```diff
@@ -0,0 +1,30 @@
+redis:
+  enabled: true
+  auth:
+    enabled: false
+  architecture: standalone
+  master:
+    persistence:
+      enabled: true
+      size: 1Gi  # Adjust the size as needed
+
+onetimesecret:
+  image: 
+    repository: onetimesecret/onetimesecret
+    tag: latest
```

Comment:
Can you set this to use: `{{ .Chart.AppVersion }}` ? This way it's defined in one spot.

Reply:
I was not able to add template directly into values.yaml so I added it from template/onetimesecret-deployment.yaml which is going to overwrite the value with appVersion.